src/bootstrap/doc.rs

```diff
@@ -491,6 +492,7 @@ impl Step for Std {
             cargo.arg("--")
                  .arg("--markdown-css").arg("rust.css")
                  .arg("--markdown-no-toc")
+                 .arg("--generate-redirect-pages")
```

Comment:
This will only apply to `core`, `alloc` and `std`. Presumably we'll want the redirects for `test` and `proc_macro` as well.